.github/workflows/karma.yml

```diff
@@ -58,8 +58,6 @@ jobs:
             - run: FORCE_NATIVE_SHADOW_MODE_FOR_TEST=1 yarn sauce:ci
             - run: DISABLE_NATIVE_CUSTOM_ELEMENT_LIFECYCLE=1 yarn sauce:ci
             - run: DISABLE_NATIVE_CUSTOM_ELEMENT_LIFECYCLE=1 DISABLE_SYNTHETIC=1 yarn sauce:ci
-            - run: API_VERSION=58 yarn sauce:ci
-            - run: API_VERSION=58 DISABLE_SYNTHETIC=1 yarn sauce:ci
```

Comment:
Looks like you are getting some unrelated changes here? Could you squash-merge or something?

Reply:
Nevermind, I gotcha: adc8d9e05